src/components/MemberCard/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import { Card, Row, Avatar } from "antd";
+import { LinkedinFilled, GithubFilled } from "@ant-design/icons";
+import "./index.css";
+
+class MemberCard extends Component {
```

Comment:
why suddenly using class components (all components in a codebase should bec onsistent). make a fn component please

Reply:
I thought since i'm rendering multiple cards and and passing a lot of props it would be cleaner to build a class and create card instances from that class. But it makes sense that all components should be consistent (thus easier read), PTAL